Backend/dev/migrations-read-only/dynamic-offer-driver-app/beckn_config.sql

```diff
@@ -41,3 +41,16 @@ ALTER TABLE atlas_driver_offer_bpp.beckn_config ADD COLUMN on_track_ttl_sec inte
 ALTER TABLE atlas_driver_offer_bpp.beckn_config ADD COLUMN on_status_ttl_sec integer ;
 ALTER TABLE atlas_driver_offer_bpp.beckn_config ADD COLUMN on_cancel_ttl_sec integer ;
 ALTER TABLE atlas_driver_offer_bpp.beckn_config ADD COLUMN on_update_ttl_sec integer ;
+
+
+
+------- SQL updates -------
+
+ALTER TABLE atlas_driver_offer_bpp.beckn_config ADD COLUMN logs_url text NOT NULL;
```

Comment:
Doesn't make sense to make this mandatory when token is optional. Make this also optional